presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -15,38 +28,67 @@ The driver is also available from Maven Central:
         <version>\ |version|\ </version>
     </dependency>
 
-Requirements
-------------
+You should be using the same version of the JDBC connector as your Presto 
+installation. If you are not on the latest release shown above, a list of all 
+currently available versions can be found in the Maven `prestosql repository 
+<https://repo.maven.apache.org/maven2/io/prestosql/presto-jdbc>`__.
 
-The JDBC driver is compatible with Java 8, and can be used with applications
-running on Java virtual machines version 8 and higher.
+Once downloaded, you must add the JAR file to the classpath of your Java 
+application. Often, this will be the lib folder of a Java application, but
+you should check the documentation for your specific application. Depending 
+on your application, you may be able to accomplish this from a dialog rather 
+than on the command line.  
+
+Once you have downloaded the JDBC driver and added it to your 
+classpath, you'll typically need to restart your application in order to 
+recognize the new driver. After that, you must register and configure the 
+driver in the application.
 
-Driver Name
+Registering
 -----------
 
-The driver class name is ``io.prestosql.jdbc.PrestoDriver``.
-Most users do not need this information as drivers are loaded automatically.
+If your application has a UI, look for a Settings option or a Drivers tab and 
+register the JDBC driver there. The steps to register the JDBC
+driver for your application on the command line depend on the specific  
+application you are using. Please check the application's documentation.
 
-Connecting
-----------
+Configuring
+-----------
 
-The following JDBC URL formats are supported:
+Along with moving the JDBC JAR file into your application's 
+classpath and registering it with the application, you must also configure 
+the following two required parameters in your Java application:
+
+* Class name: ``io.prestosql.jdbc.PrestoDriver`` 
+* JDBC URL (example): ``jdbc:presto://host:port/catalog/schema``
```

Comment:
This is redundant with the example below (which says "other ... formats" but this is not "other" since it is identical)

Reply:
removed redundant info.